db/scripts/sanitise.sql

```diff
@@ -2,12 +2,67 @@ TRUNCATE TABLE alert_runs;
 TRUNCATE TABLE emergency_login_keys;
 TRUNCATE TABLE sessions;
 
+UPDATE employments
+       SET organisation='Example Organisation',
+           job_title='Example Job Title',
+           main_duties='example@example.org';
```

Comment:
Is this one supposed to be set to an email?

Reply:
No!